src/utils/ProfileUtils.ts

```diff
@@ -54,17 +54,20 @@ async function extractConfigsToImportedProfile(
         }
 
         const progress = Math.floor(((index + 1) / zipEntries.length) * 100);
-        progressCallback(`Copying configs to profile: ${progress}%`);
+        progressCallback(progress);
     }
 }
 
 export async function installModsAndResolveConflicts(
     downloadedMods: ThunderstoreCombo[],
     profile: ImmutableProfile,
-    store: Store<any>
+    store: Store<any>,
```

Comment:
IIRC I've mentioned earlier that this helper maybe shouldn't be a part of ProfileUtils since the need to pass the Store implies we're making the coupling too tight and not getting the code reusability that's often nice for helper functions.

I think it's even more clear now that we have to pass the assignId here too. I think it's an implementation detail the helpers in this file shouldn't be concerned with. It would make sense to me to move installModsAndResolveConflicts to DownloadMixin.